forester/src/lib.rs

```diff
@@ -55,7 +55,7 @@ pub async fn run_queue_info(
         QUEUE_LENGTH
             .with_label_values(&[&*queue_type.to_string(), &tree_data.merkle_tree.to_string()])
             .set(queue_length as i64);
-        info!(
+        println!(
```

Comment:
`println!` instead of `debug` on purpose?

Reply:
Yes. It's not a debug printout, its output for the user.